readme.md

```diff
@@ -44,6 +44,46 @@ Directus is a real-time API and App dashboard for managing SQL database content.
 
 <br />
 
+## Local Development
+
+<!-- Directus API Service -->
+
+1. Run `pnpm i` at the root `directus` folder.
+2. Run `pnpm -r build` at the root `directus` folder.
+3. Navigate to `/api`
+4. Create a ENV file named `.env`
+   - You can populate the environment variables from
+     [Configuration Files](https://docs.directus.io/self-hosted/config-options.html#configuration-files)
+     - **Note:** You will need to add at the Database config variables, along with the `KEY` & `SECRET`.
+5. Run `pnpm cli database install`
+6. Run `pnpm cli database migrate:latest` to run all migrations for the database.
+7. Run `pnpm cli roles create --role Administrator --admin` to create an admin role. This will return the id value for
+   the role created.
+8. Run `pnpm cli users create --email <email> --password <password> --role <role_id>` to create a new admin user and
+   return the user id.
+   - Example:
+     `pnpm cli users create --email email@directus.io --password secret_password --role b0600d5c-b768-4bbc-a8e4-49e89d35748a `
+9. Run `pnpm dev` to start the API service.
+
+<!-- Directus Admin App -->
+
+1. Navigate to `/app`.
+2. Create a ENV file named `.env.development`.
+   - You can populate the environment variables from
+     [Configuration Files](https://docs.directus.io/self-hosted/config-options.html#configuration-files)
+3. Run `pnpm -r run build` to build all required package dependencies.
+4. Navigate to `/app` and run `pnpm dev`.
+5. Navigate to your local Directus URL, and login with the credentials you just created.
+
+### Changing Local Database with Docker
+
+- To change from a local `sqlite` database, you can use anyone of the supported databases in the root
+  [docker-compose.yml](./docker-compose.yml) by running `docker compose up -d postgres redis minio`.
+
+**Note:** If you are trying to run `postgres` on a M1/M2 MacBook (arm64), you will need to change the postgres image to
```

Comment:
Is this required? I'm on a M1 Pro and the currently provided one seems to behave just fine 🤔

Reply:
The original postgres image seems to have issues with my M1 MacBook. I found that this version significantly reduces the amount of memory required as it doesn't need to use Rosetta.

Reply:
I am happy to remove it if there is no value added.

Reply:
How about newer versions of the official `postgis` image? I like to be a bit cautious when it comes to including unknown images in the documentation